tests/test_loaders.py

```diff
@@ -1,31 +1,32 @@
 # pylint: disable=all
 import pathlib
 import pytest
+import os
 
 from uwtools.loaders import load_yaml
 
+uwtools_file_base = os.path.join(os.path.dirname(__file__))
 
 def test_yaml_loader_loads_correctly():
-    actual = load_yaml(pathlib.Path("tests/fixtures/simple.yaml"))
+    actual = load_yaml(os.path.join(uwtools_file_base,pathlib.Path("fixtures/simple.yaml")))
 
     expected = {
         "scheduler": "slurm",
         "jobname": "abcd",
-        "extra_stuff": 12345,
+        "extra_stuff": '12345',
         "account": "user_account",
-        "nodes": 1,
+        "nodes": '1',
         "queue": "bos",
-        "tasks_per_node": 4,
+        "tasks_per_node": '4',
```

Comment:
Can we not handle integers? I think it's pretty important that we can continue to handle those explicitly.

Reply:
I'm not sure what is happening with this and it definitely needs work - I'll try to make this a task to look into this discrepancy

Reply:
Added a TODO in the code comments